IdentityCore/src/webview/embeddedWebview/challangeHandlers/MSIDChallengeHandler.h

```diff
@@ -28,6 +28,9 @@
 
 + (void)handleChallenge:(NSURLAuthenticationChallenge *)challenge
                 webview:(WKWebView *)webview
+#if TARGET_OS_IPHONE
```

Comment:
nit: I'd prefer two separate methods for this one :)

Reply:
done